Assets/_BForBoss/_Analytics/Scripts/BForBossAnalytics.cs

```diff
@@ -114,5 +123,20 @@ public void EndSession()
         {
             _perigonAnalytics.EndSession();
         }
+
+        private string WorldEnumToAnalyticsValueMapper(WorldNameAnalyticsName world)
+        {
+            switch (world)
+            {
+                case WorldNameAnalyticsName.MS1_RaceCourse:
+                    return EventConstant.MS1_RaceCourse;
+                case WorldNameAnalyticsName.MS2_RaceCourse:
+                    return EventConstant.MS2_RaceCourse;
+                case WorldNameAnalyticsName.Weapons_RaceCourse:
+                    return EventConstant.Weapons_RaceCourse;
+                default:
+                    return EventConstant.Unknown;
+            }
```

Comment:
What about just calling toString on the enum? It seems a bit redundant to me to have to have both constant strings and an enum

Reply:
It's a formatting thing with snake case and lowercase

Reply:
What about just `ToString().ToLower()` then?

Reply:
Technically speaking I didn't write the enum correctly in terms of naming standard, which is why we normally map it.

It would be ms1RaceCourse

Reply:
But if someone else were to write a new course, they would know to write it in this snake case format. Since it's independent from The enum

Reply:
I see, that makes sense then

Reply:
I fixed the enums and the const variable naming